app/src/main/java/ca/bc/gov/bchealth/repository/CardRepository.kt

```diff
@@ -255,6 +191,116 @@ class CardRepository @Inject constructor(
         }
     }
 
+    private suspend fun saveCard(
+        uri: String,
+        base64EncodedPdf: String,
+        isAddingFederalTravelProof: Boolean
+    ) {
+        insert(HealthCard(uri = uri, federalPass = base64EncodedPdf), isAddingFederalTravelProof)
+    }
+
+    /*
+    * Insert or update existing health pass
+    * */
+    suspend fun insert(healthCard: HealthCard, isAddingFederalTravelProof: Boolean) {
```

Comment:
Breakdown, this method in to small methods, see sonar Qube code smell

Reply:
Taken care of this.